spec/requests/api/v0.1/trainees_spec.rb

```diff
@@ -4,17 +4,15 @@
 
 describe "Trainees API" do
   describe "GET /api/v0.1/trainees/:id" do
-    let!(:trainee) { create(:trainee, slug: "12345") }
+    let(:token) { "trainee_token" }
+    let!(:auth_token) { create(:authentication_token, hashed_token: AuthenticationToken.hash_token(token)) }
```

Comment:
can these two lines be replaced with something like `let(:token) { AuthenticationToken.create_with_random_token(provider:) } `

Reply:
We could do, but we're using both of the `let`s elsewhere in the spec. I initially had "trainee_token" written throughout the spec, but @defong suggested clearing them up with a let.